react/components/ProductPrice/index.js

```diff
@@ -128,7 +129,7 @@ class Price extends Component {
 }
 
 Price.Loader = (loaderProps = {}) => (
-  <div className={classNames('vtex-price vtex-price-loader', loaderProps.loaderClass)}>
+  <div className={classNames(productPrice.priceContainer, productPrice.priceContainerLoader, loaderProps.loaderClass)}>
```

Comment:
You're missing `productPrice.priceContainerLoader`

Reply:
I don't understand, the handle is defined